changelogs/fragments/9768-keycloak_realm-remove-id-requirement.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

I think this is more like a feature, though.
```suggestion
minor_changes:
```